README.md

```diff
@@ -169,6 +177,16 @@ $ docker run -p 9999:9999 -e JAVA_OPTS='-Dhazelcast.jmx=true -Dcom.sun.managemen
 
 Now you can connect using the address: `localhost:9999`.
 
+#### Prometheus
+
+You can use JMX Prometheus agent and expose JVM and JMX Hazelcast metrics.
+
+```
+$ docker run -p 8080:8080 -e PROMETHEUS_PORT=8080
+```
+
+Then, the metrics are available at: `http://localhost:8080/metrics`. Note that you can add also `-e JAVA_OPTS='-Dhazelcast.jmx=true'` to expose JMX Hazelcast application metrics.
```

Comment:
```
Note that you can add also `-e JAVA_OPTS='-Dhazelcast.jmx=true'` to expose JMX Hazelcast application metrics.
```

That is already mention in previous section. Do we need to say it again? It might create confussion.

Reply:
I clarified it. @mesutcelik PTAL. It's actually needed here, because in case of Prometheus, you still need to specify `-Dhazelcast.jmx=true` to see JMX metrics via Prometheus endpoints. Otherwise, Prometheus just shows the JVM metrics.